example/spec/parallel/startSF/startSfSpec.js

```diff
@@ -141,15 +168,14 @@ describe('the sf-starter lambda function', () => {
     });
 
     it('consumes the messages', async () => {
-      const response = await lambda().invoke({
+      const { Payload } = await lambda().invoke({
```

Comment:
You'll notice that I opted to not change the test expectation on line 192 of this file:

https://github.com/nasa/cumulus/pull/1062/files#diff-89bd6949a4ceda77cbb0a67687670574R192

Ultimately I just didn't like any of the possible solutions. And given that this test is not new for this PR and I'm not aware of any reported failures from CI, it seemed like it might be best to leave it alone.